src/policies/EmissionManager.sol

```diff
@@ -85,21 +89,24 @@ contract EmissionManager is IEmissionManager, Policy, RolesConsumer {
         address gohm_,
         address reserve_,
         address sReserve_,
-        address auctioneer_,
+        address bondAuctioneer_,
+        address cdAuctioneer_,
         address teller_
     ) Policy(kernel_) {
         // Set immutable variables
         if (ohm_ == address(0)) revert("OHM address cannot be 0");
         if (gohm_ == address(0)) revert("gOHM address cannot be 0");
         if (reserve_ == address(0)) revert("DAI address cannot be 0");
         if (sReserve_ == address(0)) revert("sDAI address cannot be 0");
-        if (auctioneer_ == address(0)) revert("Auctioneer address cannot be 0");
+        if (bondAuctioneer_ == address(0)) revert("Bond Auctioneer address cannot be 0");
```

Comment:
@chefomi If we plan to deploy this after the contract registry, we can simplify the deployment and maintenance of this contract by resolving the bond and CD auctioneer addresses from the contract registry when `configureDependencies()` is called. Thoughts?